### PR TITLE
Basic Sales Order

### PR DESCRIPTION
**Pull request by eduardocopat**

### CODE REVIEW

File: src/main/persistence/mongo/MongoRepository.java [hunk 1]
```diff
@@ -24,7 +24,11 @@ protected MongoRepository(String collectionName, Converter<TEntity, Document> co
     }
 
     protected MongoCollection<Document> getCollection(String name) {
-        MongoClientURI uri = new MongoClientURI(System.getenv("MONGOLAB_URI"));
+        String jesus = System.getenv("MONGOLAB_URI");
```
Comment (michelchenrich) [inline src/main/persistence/mongo/MongoRepository.java]: No Jesus here

File: src/main/domain/salesOrder/CreateSalesOrderRequest.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package main.domain.salesOrder;
+
+
+public class CreateSalesOrderRequest {
+    public String productId;
+    public int quantity;
```
Comment (michelchenrich) [inline src/main/domain/salesOrder/CreateSalesOrderRequest.java:6]: Int, Strings... meh. But Ok, I can live with that temporarily

Comment (eduardocopat) [inline src/main/domain/salesOrder/CreateSalesOrderRequest.java:6]: Yeah. I need to refactor this.

File: src/main/persistence/mongo/MongoRepository.java (hunk 1 above)
Comment (eduardocopat) [inline src/main/persistence/mongo/MongoRepository.java]: New commit abd292f fixes this.

**Pull request merged**